src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -48,7 +48,7 @@ const AppLauncher = _.flow(
       h(Fragment, [
         h(PeriodicCookieSetter, { namespace, runtimeName }),
         app === 'RStudio' && h(PlaygroundHeader, [
-          'This feature is in early development. Your files are saved on your runtime but not to your workspace. We encourage you to frequently ',
+          'This feature is in early development. Your files are saved on your cloud environment but not to your workspace. We encourage you to frequently ',
```

Comment:
Not sure how to re-phrase this,  is this still true even if you have a PD?

Reply:
I don't know if RStudio uses PD. How do we spin up RStudio?

Reply:
@sharmatime does R Studio use PD too?

Reply:
@s-rubenstein I think it's basically that an RStudio can be spun up with a custom docker image and if it's an image that relies on R then the code chooses to spin up an Rstudio environment rather than the Jupyter one. Since it gets decided in the same Cloud Environment Config i imagine it uses PD but just want to be sure